core/src/main/java/org/elasticsearch/search/aggregations/pipeline/bucketmetrics/InternalBucketMetricValue.java

```diff
@@ -68,10 +72,16 @@ public String getWriteableName() {
     }
 
     @Override
+    public double getValue() {
+        return value();
+    }
```

Comment:
Sorry if I'm asking something thats already been discussed but why are we adding this method given we already have `value()` below?